site/content/contribute/why-contribute/_index.md

```diff
@@ -0,0 +1,140 @@
+    - [Slash commands]({{< ref "/integrate/slash-commands" >}}) are apps that respond to triggers sent as messages inside Mattermost.
+    - More complicated Apps can be built as standard HTTP services in any programming language using a robust API. [Get started here]({{< ref "/integrate/apps" >}}).
+    - If you’re willing to deal with more complex development overhead, [plugins]({{< ref "/integrate/plugins" >}}) enable you to alter every component of the Mattermost experience.
+
+## You’re looking to practice your skills or give back to the community
+
+We love developers who are passionate about open-source! 
+
+If you’re looking to tackle an interesting problem, we’ve got you covered! Feel free to check out [the help wanted tickets on GitHub](https://github.com/search?utf8=%E2%9C%93&q=is%3Aopen+archived%3Afalse+org%3Amattermost+label%3A%22Help+Wanted%22++label%3A%22Up+For+Grabs%22). To take one on, just comment on the issue, and follow the process outlined in [You've found a bug](#youve-found-a-bug) of this guide.
+
+## You want to help with product translation
+
+We’re honored that you’d like to help people use Mattermost in their native language, so we treat all translators as full-fledged contributors alongside engineers and authors.
+
+Each localization community is going to have specific guidelines on how to maintain Mattermost’s distinctive voice across language barriers. Read these guides thoroughly before starting to translate:
+
+- [German](https://gist.github.com/der-test/6e04bff8a173e053811cb93e08838ca2)
+- [French](https://github.com/wget/mattermost-localization-french-translation-rules)
+- [Dutch](https://github.com/ctlaltdieliet/mattermost-localization-dutch-translation-rules)
+
+To get started:
+
+1. Join [the Mattermost translation server](http://translate.mattermost.com/) and [the localization community channel](https://community.mattermost.com/core/channels/localization) on the Mattermost Community Server.
+
+   - If your preferred language already exists on the translation server, you can start making translations or translation suggestions immediately *there on the translation server*. Don’t try to do this manually through GitHub.
+
+   - If your language is absent from the translation server, ask on the localization community channel for it to be added.
+2. Each language in whole is assigned one quality level, and with each release cycle, it can be upgraded or downgraded if necessary.
+    - **Official** — 100% of the language’s translations are verified both by a Mattermost functionality expert and by a expert speaker of the target language. The language has at least one official reviewer who updates all of the strings that have changed in the English source text, and they have successfully kept all of the translated strings updated since the last release cycle. The language has been in use for at least three full release cycles.
```

Comment:
Small change - sorry lol
But does it make sense to make the description into bullet points or line breaks for readability?
```suggestion
    - **Official** — 100% of the language’s translations are verified both by a Mattermost functionality expert and by an expert speaker of the target language. The language has at least one official reviewer who updates all of the strings that have changed in the English source text, and they have successfully kept all of the translated strings updated since the last release cycle. The language has been in use for at least three full release cycles.
```